anitya/templates/project_new.html

```diff
@@ -89,22 +93,33 @@
 <script type="text/javascript">
 
     var examples = {};
-    var more_info = {};
+    var test_regex_link = {};
     {% for plugin in plugins -%}
-    {%- autoescape false -%}
-    examples["{{ plugin.name }}"]="{{ plugin.examples | format_examples }}";
-    more_info["{{ plugin.name }}"]="{{ plugin.more_info }}";
-    {%- endautoescape -%}
+      {%- autoescape false -%}
+        examples["{{ plugin.name }}"]="{{ plugin.examples | format_examples }}";
+        console.log(examples);
+        test_regex_link["{{ plugin.name }}"]="{{ plugin.test_regex_link}}";
+
+        {% if plugin.default_regex %}
+          var _default_regex = '{{ plugin.default_regex }}';
+        {% endif %}
+      {%- endautoescape -%}
     {%- endfor %}
 
   function show_hide(){
     $('#regex_row').hide();
+    $('#default_regex_row').hide();
     $('#version_url_row').hide();
     $('#insecure_row').hide();
     $('#regex').attr('placeholder', '');
 
     $("label[for='version_url']").text('Version URL');
     var str = $( '#backend option:selected' ).text();
+
+    if (str != 'Sourceforge') {
+      $('#sourceforge_version_url_instr').hide();
+    }
```

Comment:
This should be un-necessary, the text after the version_url should disapear if the backend != SourceForge